content/applicants/student-life.md

```diff
@@ -0,0 +1,20 @@
+title: О студенческой жизни
+slug: student-life
+category: applicants
+tags: [_nav,_ignore_nav]
+submenu_index: 1
```

Comment:
Since we're not going to display a link to the page in navigation bar, set its `submenu_index` to `None` in order not to confuse anyone.